robot/EDA/resources/locators.py

```diff
@@ -95,6 +96,14 @@
         "settings_tab": "(//li[@class='slds-tabs__item slds-text-heading--label slds-active' and @role='tab' and @title='Settings'])[1]/a[contains(text(),'Settings')]",
         "affl_mappings_tab": "//a[contains(text(),'Affiliation Mappings')]",
     },
+    "eda_settings_cc": {
+        "default_cc_checkbox": "//div[text()='Enable Course Connections']/following-sibling::div/descendant::img",
+        "dropdown_values": "//div[text()='{}']/following-sibling::div/select/option[text()='{}']",
+        "dropdown_values_count": "//div[text()='{}']/following-sibling::div/select/option",
+        "enable_cc_checkbox": "//div[text()='Enable Course Connections']/following-sibling::div[1]/descendant::span",
```

Comment:
can u use something else instead of indexing? also this locator looks almost similar to line 100 can this be parameterized and reused something like //div[text()='{}']/following-sibling::div[some attribute]/descendant::{}

Reply:
Index wasn't really required there anyway. Removed it